frontend-cert/js-projects/first-oop-lab/user-stories.md

```diff
@@ -0,0 +1,25 @@
+You should define a constant of type ENUM named `projectStatus` with the different project statuses you want to have such as `PENDING`, `SUCCESS`, or `FAILURE` with ypur preferred descriptions.
```

Comment:
the concept of ENUM hasn't really be explained at this point.
that won't really happen until the typescript section.

so this will need to be reworded to just have them create the object with the properties and value you want them to create